console_conf/controllers/identity.py

```diff
@@ -156,9 +168,21 @@ def make_ui(self):
             device_owner = get_device_owner()
             if device_owner:
                 self.model.add_user(device_owner)
-                key_file = os.path.join(device_owner["homedir"], ".ssh/authorized_keys")
-                cp = run_command(["ssh-keygen", "-lf", key_file])
-                self.model.user.fingerprints = cp.stdout.replace("\r", "").splitlines()
+                # if we are in strict snap confinement we cannot
+                # access random user's .ssh dir
+                if os.getenv("SNAP_CONFINEMENT", "classic") == "strict":
```

Comment:
What should the login view look like in this case?  It seems to quite strongly assume that we can show fingerprints, just omitting them isn't great.

Reply:
I think this isn't a problem anymore. I reviewed the code and how User is used throughout the places here and in the login view and none of the places reference `fingerprints`, which means they were never shown and were collected for no clear reason. At this point I think we can just drop it and we'll have one less problem to address. I'll propose a separate branch with the cleanup.

Reply:
opened https://github.com/canonical/subiquity/pull/1905